README.md

```diff
@@ -30,32 +30,48 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. Describe the biggest difference between `.forEach` & `.map`.
 
+    .forEach calls the given function on each element of the object/array, .map will also do this for each element but will capture return values in a new array.
+
```

Comment:
The answer are right,  .forEach mutates the original array, map creates a new array